alws/dramatiq/build.py

```diff
@@ -192,7 +203,18 @@ async def _build_done(request: build_node_schema.BuildDone):
         )
 
         if all_build_tasks_completed:
-            build_id = await _get_build_id(db, request.task_id)
+            failed_tasks = [task for task in build_tasks if task.status == BuildTaskStatus.FAILED]
+            if len(failed_tasks) == len(build_tasks):
+                await db.execute(
```

Comment:
Not sure that we need to update these fields via query because we already have build tasks from DB, we can set `build_srpm_url=None` using object attributes
```python
for task in build_tasks:
    task.build_srpm_url = None
```